### PR TITLE
etcdmain: handle TLS in grpc-proxy listener

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

**Comment by gyuho** [comment]
@xiang90 Please review. Confirmed that it works.

File: etcdmain/grpc_proxy.go [hunk 1]
```diff
@@ -119,7 +120,6 @@ func startGRPCProxy(cmd *cobra.Command, args []string) {
 	pb.RegisterLeaseServer(server, leasep)
 	pb.RegisterMaintenanceServer(server, mainp)
 	pb.RegisterAuthServer(server, authp)
-	grpc_prometheus.Register(server)
```
Comment (heyitsanthony) [inline etcdmain/grpc_proxy.go]: should there be a `mux.Handle("/metrics", prometheus.Handler())` or something to replace this?

Comment (heyitsanthony) [inline etcdmain/grpc_proxy.go]: nm, I see it's in the code below...

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**